tofu/geom/_GG.pyx

```diff
@@ -925,218 +883,366 @@ def discretize_vpoly(double[:,::1] VPoly, double dL,
 #                           i.e. Discretizing Volumes
 #
 # ==============================================================================
-def _Ves_Vmesh_Tor_SubFromD_cython(double dR, double dZ, double dRPhi,
+def _Ves_Vmesh_Tor_SubFromD_cython(double rstep, double zstep, double phistep,
```

Comment:
Ok,
However, the point in the name dRPhi was that it made it explicit that this resolution (or step) was a distance (angle*radius), not an angle.
The choice of name phistep does not make this clear, since r does not appear in it
Is it a lot of work to rename to something closer to a distance: rphistep ?